.github/workflows/devops-boards.yaml

```diff
@@ -18,6 +18,7 @@ permissions:
 
 jobs:
   ado:
+    if: ${{ !github.event.issue.pull_request }}
```

Comment:
does this work? cant find documentation for this on first look

Reply:
it's used in the sample I referenced here: https://github.com/danhellem/github-actions-issue-to-work-item?tab=readme-ov-file#examples

but taking a second look, it seems like we shouldn't use this filter anyway as we'd probably still like the workflow to trigger if a PR actioned on the issue. i'll remove the conditional.